### PR TITLE
Migration to metal stack

### PR DESCRIPTION
**Pull request by Gerrit91**
This PR completes the migration to metal-stack.

- [x] Renames module
- [x] Adds Github Actions workflows
- [x] Adapts documentation

### CODE REVIEW

File: docker-make.yml [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-  - HTTPS_PROXY=${HTTP_PROXY}
-  - NO_PROXY=${NO_PROXY}
-  - no_proxy=${NO_PROXY}
+username: metalstack
+registry-host: docker.io
+after:
+  - mkdir -p tmp
+  - mkdir -p result
+  # the second latest built container is the builder, which contains the platform binaries
+  - docker cp $(docker create $(docker image ls --format '{{ .ID }}' | sed -n 2p)):/work/bin tmp
```
Comment (majst01) [inline docker-make.yml]: hu, what is this

**Comment by majst01** [comment]
ready to merge ?

**Comment by majst01** [comment]
No

**Comment by Gerrit91** [comment]
Now I am ready.

File: docker-make.yml (hunk 1 above)
Comment (Gerrit91) [reply docker-make.yml]: That's copying the binaries on to the Github runner.

Comment (Gerrit91) [reply docker-make.yml]: Do you have a better idea? We need to somehow get the artifacts from the builder container.

Comment (Gerrit91) [reply docker-make.yml]: We can handle this in a different PR. This is the first version that kind of works.

**Pull request merged**